qt/widgets/common/src/FunctionBrowser.cpp

```diff
@@ -2262,16 +2279,31 @@ void FunctionBrowser::updateMultiDatasetParameters(
         throw std::invalid_argument("Function has incorrect number of domains");
       }
       // update function
+      {
+        auto sfun = multiFun->getFunction(0);
+        const auto globalParameters = getGlobalParameters();
+        for (int j = 0; j < globalParameters.size(); ++j) {
+          auto paramName = globalParameters[j];
```

Comment:
`QString` has copy-on-write mechanism, but would it still be worth declaring this variable `auto const &`?